appcenter-data/__tests__/Data.js

```diff
@@ -77,12 +77,13 @@ describe('App Center Data read operation tests', () => {
 });
 
 describe('App Center Data list operation tests', () => {
-  test('list is called ', async () => {
+  test('list is called with partition and TimeToLive.DEFAULT', async () => {
```

Comment:
As mentioned in https://github.com/microsoft/appcenter-sdk-react-native/pull/767/files#r344940943, there is a gap in the test coverage. `Data.list` API takes partition and readOptions as parameters but we don't have a test case covering list API called with both parameters.